Src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/ExpectedOutputs/SEC101_036.MySqlConnectionString.sarif

```diff
@@ -155,49 +155,6 @@
           },
           "rank": 44.6
         },
-        {
-          "ruleId": "SEC101/036",
-          "ruleIndex": 0,
-          "level": "note",
-          "message": {
-            "id": "Default",
-            "arguments": [
-              "passwo…",
-              "an apparent ",
-              "",
-              "ADO or ODBC MySQL connection string",
-              "",
-              " (no validation occurred as it was not enabled. Pass '--dynamic-validation' on the command-line to validate this match)"
-            ]
-          },
-          "locations": [
-            {
-              "physicalLocation": {
-                "artifactLocation": {
-                  "uri": "src/Plugins/Tests.Security/TestData/SecurePlaintextSecrets/Inputs/SEC101_036.MySqlConnectionString.ps1",
-                  "uriBaseId": "SRC_ROOT"
-                },
-                "region": {
-                  "startLine": 3,
-                  "startColumn": 122,
-                  "endLine": 3,
```

Comment:
>": [](http://example.com/codeflow?start=26&length=2)

Nice! now we don't have two candidates, one with a port and one without! In the future, you could explain this code change to me. i.e., we should always try to explain to each other why things appeared or disappeared if it's not obvious in the baseline logs.